src/components/lib/esmap.js

```diff
@@ -145,9 +145,6 @@ function renderEdges(g, data, ref, layerId) {
       var layerClass = ' l'+layerId;
       return 'edge edge-az edge-az-' + name + connections + layerClass;
     })
-    .attr('text', function (d) {
```

Comment:
I'm not 100% on why this was deleted, I have a feeling this will break something somewhere unexpectedly.